packages/mdc-textfield/README.md

```diff
@@ -46,6 +46,18 @@ npm install @material/textfield
 </div>
 ```
 
+#### Helper Text
```

Comment:
I think these should be under variants?

Reply:
I thought that helper text and icons were sub components (not variants)...Maybe they could be their own `sub component` section?

Reply:
I checked with Bonnie and we're thinking it's OK under Variants rather than inventing another section. If it'd help, maybe update the headings to "Text Field with Helper Text" and "Text Field with Leading/Trailing Icons"?

Reply:
Ya that makes more sense to me. Will make that update